content/api/integrations/code_snippets/api-integrations-pagerduty.sh

```diff
@@ -16,6 +17,38 @@ curl -v -X PUT -H "Content-type: application/json" \
   }' \
 "https://api.datadoghq.com/api/v1/integration/pagerduty?api_key=${api_key}&application_key=${app_key}"
 
-curl -v "https://api.datadoghq.com/api/v1/integration/pagerduty?api_key=${api_key}&application_key=${app_key}"
+# Update an existing configration.
+# In this example, we are overriding `subdomain`, `api_token` and `schedules` values
+# with new values.
+curl -v -X PUT -H "Content-type: application/json" \
+-d '{
+      "subdomain": "<PAGERDUTY_SUB_DOMAIN>",
+      "schedules": ["<SCHEDULE_1>", "<SCHEDULE_2>"],
+      "api_token": "<PAGERDUTY_TOKEN>"
+  }' \
+"https://api.datadoghq.com/api/v1/integration/pagerduty?api_key=${api_key}&application_key=${app_key}"
 
-curl -v -X DELETE "https://api.datadoghq.com/api/v1/integration/pagerduty?api_key=${api_key}&application_key=${app_key}"
\ No newline at end of file
+# Append to the `services` and `schedules` list.
```

Comment:
"Add new services and schedules to the existing configuration" instead? thoughts?